TuSimple/naive-ui
# feat(drawer): add rounded corner theme variable (#4696)

# PR: feat(drawer): add rounded corner theme variable (#4696)

## Description
Introduce a new theme variable to control the rounded corner radius for the Drawer component. This enhancement aims to provide greater customization and flexibility in styling the Drawer, allowing developers to easily adjust the rounded corner appearance based on their design preferences.

## Changes Made
- Added a new theme variable to control the rounded corner radius.
- Implemented the variable in the Drawer component for consistent styling.

## How to Test
1. Ensure the latest changes are pulled from the branch.
2. Apply the new theme variable to customize the rounded corner radius in the Drawer component.
3. Verify the changes visually and confirm the expected styling.

Link: (#4696)
